pkg/controller/common/util.go

```diff
@@ -211,6 +212,8 @@ const (
 
 	//AnnDefaultStorageClass is the annotation indicating that a storage class is the default one.
 	AnnDefaultStorageClass = "storageclass.kubernetes.io/is-default-class"
+	// AnnDefaultVirtStorageClass is the annotation indicating that a storage class is the default one for virtualization purposes
+	AnnDefaultVirtStorageClass = "storageclass.kubevirt.io/is-default-class"
```

Comment:
Maybe this should be storageclass.kubevirt.io/is-default-virt-class to avoid confusion with the standard k8s annotation.  I know we have a different URI but still.

Reply:
Was just following the pattern
`snapshot.storage.kubernetes.io/is-default-class`
`ingressclass.kubernetes.io/is-default-class`
`storageclass.kubernetes.io/is-default-class`

Reply:
I am up for extra clarity though so just let me know if this still makes sense

Reply:
Done